source/simulator/parameters.cc

```diff
@@ -757,20 +757,26 @@ namespace aspect
                          Patterns::Integer (0),
                          "The number of adaptive refinement steps performed after "
                          "initial global refinement but while still within the first "
-                         "time step.");
+                         "time step. These refinement steps (n) are added to the value "
+                         "for initial global refinement (m) so that the final mesh has "
+                         "cells that are at most on refinement level $n+m$   ");
```

Comment:
Please remove the trailing spaces and add a period: `$n+m$.");`

Reply:
Fixed and pushed commit to master.

Reply:
Thanks, good to go.